### PR TITLE
Add a README.md file

### LINKED ISSUE #3: Add README and examples

### PR DESCRIPTION
**Pull request by majecty**
This PR closes #3.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+CodeChain RPC JS
+=================
+
+CodeChain RPC JS is a javascript library that calls RPC to a CodeChain node.
```
Comment (ScarletBlue) [inline README.md]: CodeChain RPC JS is a JavaScript library that calls RPC to a CodeChain node.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+How to install
+--------------
+
+If you use NPM, use the command below.
```
Comment (ScarletBlue) [inline README.md]: If you are using NPM, use the command below:

File: README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+```
+npm install codechain-rpc --save
+```
+
+If you use Yarn, use the command below.
```
Comment (ScarletBlue) [inline README.md]: If you prefer Yarn, use the command below:

**Review by ScarletBlue — CHANGES_REQUESTED** [review]

**Comment by majecty** [comment]
@ScarletBlue I fixed them.

**Review by ScarletBlue — APPROVED** [review]

**Pull request merged**